certified-connectors/FactSet/apiDefinition.swagger.json

```diff
@@ -1,1411 +1,1422 @@
+                            },
+                            "additionalProp3": {
+                              "type": "string",
+                              "description": "additionalProp3"
+                            }
+                          },
+                          "description": "regionalPermId"
+                        }
+                      },
+                      "description": "idResolutions"
+                    },
+                    "dateRange": {
+                      "type": "object",
+                      "properties": {
+                        "created": {
+                          "type": "object",
+                          "properties": {
+                            "gt": {
+                              "type": "string",
+                              "description": "gt"
+                            }
+                          },
+                          "description": "created"
+                        },
+                        "updated": {
+                          "type": "object",
+                          "properties": {
+                            "gt": {
+                              "type": "string",
+                              "description": "gt"
+                            }
+                          },
+                          "description": "updated"
+                        }
+                      },
+                      "description": "dateRange"
+                    }
+                  },
+                  "description": "meta"
+                },
+                "errors": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "id"
+                      },
+                      "code": {
+                        "type": "string",
+                        "description": "code"
+                      },
+                      "title": {
+                        "type": "string",
+                        "description": "title"
+                      },
+                      "detail": {
+                        "type": "string",
+                        "description": "detail"
+                      },
+                      "source": {
+                        "type": "object",
+                        "properties": {
+                          "parameter": {
+                            "type": "string",
+                            "description": "parameter"
+                          }
+                        },
+                        "description": "source"
+                      }
+                    }
+                  },
+                  "description": "errors"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Signals - Headlines",
+        "operationId": "get-headlines",
```

Comment:
@KyleVErwin all operationIds should be in Camel Case. is it possible to change to "GetHeadlines". 
Please do not change the operationId for the existing operations, only for the new ones. As if you change an existing operationId will break existing flows.